conf/default.toml

```diff
@@ -71,6 +71,11 @@ background_color = "#fdfbff"
 # This mandatory parameter indicates which background should be used for the main header
 header_background = "center / cover url('assets/img/default_header_bg.webp')"
 
+# This mandatory parameter indicates which fallback image should be used for thumbnails, 
```

Comment:
I don't think this parameter should be mandatory actually, and I also realized that the header background above shouldn't be mandatory as well; this is justified by the fact that we then won't be able to update the default placeholder (or the default header) in subsequent versions without asking users to update their configs manually.

Maybe we made too many config parameters mandatory in the first place?

Reply:
Ok, I agree, I'll rework this

Reply:
I adapted the param to be optional in the config, providing a default placeholder in the `app-config` if necessary and removing the embedded SVG "loading placeholder" to be consistent.